exercise-002/main.cpp

```diff
@@ -6,29 +6,57 @@
 
 auto main(int argc, char **argv) -> int
 {
+    std::srand(std::time(nullptr));
     /**
      * CLI11 is a command line parser to add command line options
-     * More inpo at https://github.com/CLIUtils/CLI11#usage
+     * More info at https://github.com/CLIUtils/CLI11#usage
      */
+    int count =20;
     CLI::App app{PROJECT_NAME};
     try
     {
         app.set_version_flag("-V,--version", fmt::format("{} {}", PROJECT_VER, PROJECT_BUILD_DATE));
+        app.add_option("-c,--count", count, "Variable count");
         app.parse(argc, argv);
     }
     catch (const CLI::ParseError &e)
     {
         return app.exit(e);
     }
-
+    fmt::print("Count: {}",count);
     /**
      * The {fmt} lib is a cross platform library for printing and formatting text
      * it is much more convenient than std::cout and printf
      * More info at https://fmt.dev/latest/api.html
      */
     fmt::print("Hello, {}!\n", app.get_name());
-
+    std::vector<int> data(count);
+    fmt::print("Created vector of size {} \n", data.size());
     /* INSERT YOUR CODE HERE */
-
+    for (int i = 0; i < data.size(); i++)
+    {
+        data[i]=rand() %101;
```

Comment:
```suggestion
        data[i]=(rand() %100)+1;
```
This will generate numbers in the range from `1...100`